pyiron_atomistics/lammps/base.py

```diff
@@ -583,7 +586,7 @@ def collect_output_log(self, file_name="log.lammps", cwd=None):
                         ((df.columns.str.len() == 3) & df.columns.str.startswith("P"))
                     ]
                 )
-                df["pressures"] = pressures.tolist()
+                pressure_dict["pressure"] = pressures
```

Comment:
`pressures` to `pressure`?

This being said, I personally prefer `pressure` to `pressures`, because except for `cells` and `pressures`, per-atom quantities are always in plural and global quantities are in singular in pyiron.

Reply:
I too prefer `pressure` since this is a per box quantity but I now realize that all codes use `pressures`. Better to make the transition now?

Reply:
Ok done!